index.ts

```diff
@@ -286,28 +286,40 @@ TEST: addTests('isQuickPR', [
 	'https://github.com/sindresorhus/refined-github/compare/test-branch?quick_pull=1',
 ]);
 
-const prStateSelector = [
+const stateSelector = [
 	'.State',
 	'[class^="StateLabel"]',
```

Comment:
Maybe use this is better.

<img width="1252" alt="image" src="https://github.com/user-attachments/assets/39d3c3b7-d9af-4401-bf3e-4f69404d4e8b">